libsolutil/InvertibleMap.h

```diff
@@ -33,26 +33,29 @@ struct InvertibleMap
 
 	void set(K _key, V _value)
 	{
-		if (values.count(_key))
-			references[values[_key]].erase(_key);
-		values[_key] = _value;
+		auto it = values.find(_key);
```

Comment:
Is the expectation that this is faster because it stops at first encounter?

Reply:
It does not have to do the lookup twice.